.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+> __ALL Pull-Requests require an associated [Issue](https://github.com/thegooddocsproject/templates/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) as documented in our [Contributing Guide](https://github.com/thegooddocsproject/templates/blob/master/contribute.md#contributing).__
+
+## Purpose / Why
+
+> _**Replace This Text** - Restate the purpose/jusitifcation of this work and include links to the Issue inside of the purpose/why._
+
+## What Changes Were Made?
+
+> _**Replace This Text** - State clearly the direct additions or modifications made in this change-request._
+
+## Verification
+
+> _**Replace This Text** - Document the details that help a reviewer understand what steps or actions should be taken in-order to verify the work is completed and satisfies the initial scope of work._
+
+---
+
+## Type of Change
+
+> __Pull-Request Opener__ should select one of the following
+
+* [ ] __Template Updates__ - _style_, _grammar_, _content_
+* [ ] __Template Structuring__ - _moving template files and or folders_
+* [ ] __Tooling & Support__ - _updates to the repository tooling_
+
+---
+
+## Checklist
+
+> __Pull-Request Reviewer__ should ensure the following
+
+* [ ] Are Issues Linked Correctly?
+* [ ] If, template updates do they align with [Google StyleGuide]()?
+* [ ] Did the PR receive :+1: from core-maintainers?
```

Comment:
Check to see what roles we define for our project. Right now, the only official role we have is a PSC member. (This will likely be extended soon.)

I think we need to quantify support from core maintainers:
[ ] Did the PR receive at least one :+1: and no :-1: from core-maintainers?

Reply:
Making your line suggestion, we can always update this for new roles.